src/components/songs/list/index.jsx

```diff
@@ -9,7 +7,7 @@ export const SongList = () => {
             {/* header */}
             <div className="font-bold text-xl flex justify-between items-center p-6 border-b border-gray-200">
                <span>Songs</span>
-               <FontAwesomeIcon icon={faHeadphonesAlt} style={{ color: '#77db89' }} />
+               <i className="fa-solid fa-headphones-alt" style={{ color: '#77db89' }}></i>
```

Comment:
We are likely to use this color for different icons at different locations in the app, is there a way to store the color value in tailwind? So that we can grab the color from there.

https://tailwindcss.com/docs/customizing-colors